Sources/Common/Background Queue/QueueRunner.swift

```diff
@@ -13,20 +13,13 @@ public protocol QueueRunner: AutoMockable {
 
```

Comment:
This file contains the most changes for the PR. I moved code from Tracking module's QueueRunner into this one. Now, all of the Tracking API HTTP calls are in 1 module (which, I think is a good thing thinking about modularity in the code base).